helm-app/templates/phpmyadmin-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.phpmyadmin.service.name }}
+  namespace: {{ .Values.phpmyadmin.namespace }}
+spec:
+  type: {{ .Values.phpmyadmin.service.type }}
+  selector:
+    app: {{ .Values.phpmyadmin.name }}
+  ports:
+    - protocol: TCP
+      port: {{ .Values.phpmyadmin.service.port }}
+      targetPort: {{ .Values.phpmyadmin.service.targetPort }}
+      nodePort: {{ .Values.phpmyadmin.service.nodePort }}

```

Comment:
If I keep the type as `headless` or `ClusterIP`, what would the value of `nodePort` be?

Reply:
Fixed it to be suitable for both.